.github/workflows/test.yml

```diff
@@ -1,6 +1,9 @@
 name: e2e-tests
 
-on: push
```

Comment:
Please remember to restore to the original version

Reply:
Done